automotive/build.gradle.kts

```diff
@@ -1,33 +1,21 @@
 import com.google.gms.googleservices.GoogleServicesPlugin.GoogleServicesPluginConfig
 
 plugins {
-    alias(libs.plugins.android.application)
-    alias(libs.plugins.kotlin.android)
-    alias(libs.plugins.ksp)
-    alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.homeassistant.android.application)
     alias(libs.plugins.google.services)
-    alias(libs.plugins.hilt)
-    alias(libs.plugins.compose.compiler)
+    alias(libs.plugins.homeassistant.android.dependencies)
 }
 
 android {
-    namespace = "io.homeassistant.companion.android"
-
-    compileSdk = libs.versions.androidSdk.compile.get().toInt()
-
-    ndkVersion = "21.3.6528147"
-
     useLibrary("android.car")
 
     defaultConfig {
-        applicationId = "io.homeassistant.companion.android"
+        // Override minSDK since automotive bellow 29 does not exist
         minSdk = libs.versions.androidSdk.automotive.min.get().toInt()
-        targetSdk = libs.versions.androidSdk.target.get().toInt()
-
-        versionName = project.version.toString()
-        // We add 2 because the app, wear (+1) and automotive versions need to have different version codes.
-        versionCode = (System.getenv("VERSION_CODE")?.toIntOrNull() ?: 1) + 3
```

Comment:
It was saying 2 put actually it add 3. I didn't want to break anything so I kept 3 and updated the doc.